### ISSUE TITLE
icon for meeting in depth bar

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#827]
the icon for a meeting in the depth bar (& maybe elsewhere?) is the missing image icon of the shrugging person. this is a bit confusing, especially if you have quite a few things in the depth bar. could we find a better icon - maybe the same as what riad suggested for the icon in the left hand tool menu? (i can't remember where that is now but it is somewhere ... )
![Screenshot from 2022-03-09 15:29:21](https://user-images.githubusercontent.com/3437324/157462074-54504023-cc07-4596-ac84-faa1b71bbf02.png)

**Comment by riadsalameh** [comment]
Hey Helen, Sorry for the extremely late reply. What does the shrugging option do? And what was the icon that I had suggested? I can work on something new also

**Comment by elisadcg** [comment]
The original icon is here (svg file)
[](url
![icon-meeting](https://user-images.githubusercontent.com/7611913/160825963-8a5e2b7e-70eb-44d9-b405-618a9a368f97.svg)
)

**Comment by nguyenhongphat0** [comment]
I didn't notice that there is a relationship between the 2 tools 😆 It's look like this at the moment: 

<img width="335" alt="Screen Shot 2022-03-31 at 00 10 10" src="https://user-images.githubusercontent.com/29925961/160894288-53416bac-ef46-4789-8996-beb9f4604575.png">

**Comment by helenvarley** [comment]
@riadsalameh the shrugging icon is a generic icon used to indicate that a graphic file is missing. what @nguyenhongphat0 has shown above is a good solution - it's clear that it's the meeting, and the name of the meeting shows in case you have more than one meeting open (altho i can't think of a scenario where you would want more than one jitsi meeting, as then everyone would be in 2 meetings ... :laughing: )

**Comment by vickis** [comment]
Yes it has the different icon now briefly it appeared to have same as
streaming icon :)

**Comment by vickis** [comment]
hmm yes or could use the little jitsi Aladdin
because that is all that's what we have on the stage ...

[image: image.png]

**Comment by helenvarley** [comment]
actually we also need an icon for the individual jitsi window in the Depth toolbar. this is still showing as the missing image icon (shrugging silhouette) and there is no name on rollover, so if several people have their individual jitsi cam on the stage it's not so easy to see which is which (if you rollover then of course the frame appears, but it would be helpful to have the name as well).

it should be ok to have the same antenna icon that we have for the meeting, as long as the name also appears. unless someone else has a better idea?

![Uploading Screenshot from 2022-10-12 14:49:13.png…]()

**Comment by vickis** [comment]
It was showing as meeting icon for me in chrome …

**Comment by vickis** [comment]
Yep fine for me chrome and firefox :) Mac 10.13.6
<img width="827" alt="image" src="https://user-images.githubusercontent.com/4249450/195828859-de199a81-9d7d-4ced-9a4b-231848452c2c.png">
<img width="893" alt="image" src="https://user-images.githubusercontent.com/4249450/195829817-1eb9fde4-8ab1-4ca3-95eb-8aa38fb7355a.png">

**Comment by helenvarley** [comment]
not talking about the jitsi meeting icon, it's the individual jitsi icon that is showing as a broken image - see my screenshots above

**Comment by vickis** [comment]
Ah I see - yes that is happening for me too :)
